pod.go

```diff
@@ -152,6 +155,18 @@ func (pc *podCollector) Describe(ch chan<- *prometheus.Desc) {
 	ch <- descPodContainerResourceLimitsMemoryBytes
 }
 
+func printControllers(annotation map[string]string) string {
+	value, ok := annotation[api.CreatedByAnnotation]
+	if ok {
+		var r api.SerializedReference
+		err := json.Unmarshal([]byte(value), &r)
```

Comment:
@fabxc thank you for your response! Initially,I also confused with `controllers`, but in the returned result of `kubectl describe po pod-name` [describe code](https://github.com/kubernetes/kubernetes/blob/c38717b73ae5c35f91d2e30bf0470575c09ad9d0/pkg/printers/internalversion/describe.go#L541),according to [code](https://github.com/kubernetes/kubernetes/blob/1d97472361e61106a1da1ce33da97f243e4d0441/pkg/client/listers/core/internalversion/replicationcontroller_expansion.go#L37),controllers refer to which one actually manage the pod